persistence/src/commonMain/db_user/migrations/51.sqm

```diff
@@ -1,167 +1,87 @@
-import com.wire.kalium.persistence.dao.QualifiedIDEntity;
-import com.wire.kalium.persistence.dao.message.MessageEntity.ContentType;
-import com.wire.kalium.persistence.dao.message.MessageEntity.MemberChangeType;
-import com.wire.kalium.persistence.dao.message.MessageEntity;
-import com.wire.kalium.persistence.dao.message.MessageEntityContent;
-import com.wire.kalium.persistence.dao.message.RecipientFailureTypeEntity;
-import kotlin.Int;
-import kotlin.Boolean;
-import kotlin.collections.List;
-import kotlinx.datetime.Instant;
+DROP VIEW IF EXISTS ConversationDetails;
 
-CREATE TABLE ButtonContent (
```

Comment:
I see that ButtonContent and MessageDetailsView is entirely removed from migration, is this an expected action?

Reply:
i already fixed the migration, the cherry pick action did overwrite the old migration file